ControllerKey.js

```diff
@@ -116,18 +116,19 @@ export class ControllerKey {
    * wrap or unwrap keys.
    *
    * If this ControllerKey is a controller of the KEK, then the API for it can
-   * returned by passing only the key description. Otherwise, an OCAP-LD
-   * authorization capability must also be passed; without this capability,
-   * calls to the returned API will not be authorized to perform KEK operations.
+   * be returned by passing only the key id and type.
+   * Otherwise, an OCAP-LD authorization capability must also be passed;
```

Comment:
Either add whitespace above this or merge into the above paragraph.

Reply:
Merged into a single paragraph here: https://github.com/digitalbazaar/web-kms-client/pull/2/commits/26ca0d200d9e088e209e8a2dd1982b1f4de56bec